DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -577,7 +509,7 @@ void Converter<DDLElementaryMaterial>::operator()(xml_h element) const {
                "+++ Compared to XML values: Atomic weight %g, Atomic number %u",
                atomicWeight,
                atomicNumber);
-      static const double weightTolerance = 1.0e-6;
+      static constexpr double const& weightTolerance = 1.0e-6;
```

Comment:
Likely better without the reference
```suggestion
      static constexpr double const weightTolerance = 1.0e-6;
```

Reply:
yes, I was hesitating about it... the object has static storage duration due to life extension by a static reference